articles/other/other-ref-browsers.md

```diff
@@ -23,28 +23,19 @@ This document outlines the browser requirement for UKCloud services.
 
 ## UKCloud for VMware
 
-For UKCloud for VMware, we recommend that you use the Mozilla Firefox Extended Support Release (ESR 52) 32-bit browser. This browser is supported by both the UKCloud Portal and by the Web Console for vCloud Director 8.20 (the version used by UKCloud for VMware).
+For UKCloud for VMware, vCloud Director 9.1 is compatible with many current and last major browser revisions.
+Version 9.1 of vCloud Director was tested by VMware and supports:
 
-Firefox ESR 52 is an older version of Firefox, updated with new security patches, that remains compatible with the plugin type that the vCloud Director Web Console uses on our platform. You can find Firefox ESR at:
+- Google Chrome 64
```

Comment:
Add an empty line after each bullet point so that they are rendered correctly in the generated HTML